### ISSUE TITLE
Robust coloring metrics

### ISSUE THREAD

**Issue by mortonjt** [biocore/emperor#708]
It maybe super useful to enable quartiles / midpoint normalization to allow for better color discrimination.  The [seaborn.heatmap](https://seaborn.pydata.org/generated/seaborn.heatmap.html) function has a robust keyword is a good reference for coloring via quartiles.  And see [here](https://matplotlib.org/users/colormapnorms.html#custom-normalization-two-linear-ranges) for midpoint normalization.

A real reason why this is useful is because a bad coloring scheme can hide real details.  Below is an example where I have enabled continuous values

![image](https://user-images.githubusercontent.com/4184797/51341783-24514800-1a60-11e9-96bb-a3011f999dab.png)

And below is where I disabled continuous values.

![image](https://user-images.githubusercontent.com/4184797/51341791-27e4cf00-1a60-11e9-9c87-fc1f416eac98.png)

There are clear trends and more could be leveraged to properly evaluate the continuous value coloring scheme.

**Comment by mortonjt** [comment]
@ElDeveloper would there be any interest in having this as a contribution?  Right now, the continuous values option isn't being used very often because of the extreme outliers.

Furthermore, the midpoints are not being properly set, so colors are often skewed towards one end of the spectrum.  Maybe having 3 text boxes for vmin, midpoint and vmax should do the trick.  Thoughts?

**Comment by ElDeveloper** [comment]
@mortonjt, yes this would be very handy to have in as a contribution! I like the idea of the three text boxes. These could show up in the UI once the continuous selection has been enabled. Also, handling outliers would be excellent! Let me know if you need some help with this.
